linode/manager
# refactor: [M3-6901, M3-6918] - Replace `react-select` with `Autocomplete` in Domains

## Description 📝
We want to get rid of our dependency on react-select for accessibility reasons and to consolidate our usage of third-party libraries.

## Changes  🔄
List any change relevant to the reviewer.
- Replace `react-select` with `Autocomplete` component within the Domains feature.

## Target release date 🗓️
22nd July, 2024

## How to test 🧪

### Verification steps
(How to verify changes)

- Navigate to http://localhost:3000/domains/create
  - Verify all the dropdown functionality in `CreateDomain` component continue to work as expected.
- Navigate to http://localhost:3000/domains/:id
   - Verify all the dropdown functionality in `DomainRecordDrawer` component continue to work as expected.

## As an Author I have considered 🤔

*Check all that apply*
